runtime/kusama/src/xcm_config.rs

```diff
@@ -428,6 +433,63 @@ impl pallet_xcm::Config for Runtime {
 	type AdminOrigin = EnsureRoot<AccountId>;
 }
 
+fn derive_tinkernet_multisig(id: u128) -> AccountId {
+	AccountId::decode(&mut sp_runtime::traits::TrailingZeroInput::new(
+		&(
+			sp_core::H256([
+				212, 46, 150, 6, 169, 149, 223, 228, 51, 220, 121, 85, 220, 42, 112, 244, 149, 243,
+				80, 243, 115, 218, 162, 0, 9, 138, 232, 68, 55, 129, 106, 210,
+			]),
+			id as u32,
```

Comment:
no overflow handling?

Reply:
While the id should technically never be over u32::MAX, you're right that it should be handled here as Kusama/Rococo shouldn't have to trust that to be true. Added overflow handling!